packages/codefixes/test/ts-codefixes.test.ts

```diff
@@ -244,9 +244,9 @@ describe('ts-codefixes', () => {
       'requireInTs/a.ts',
     ]);
   });
-   */
 
   test('unusedIdentifier', async () => {
```

Comment:
instead of commenting this out. lets test.skip and leave a comment above the test as to why we are skipping